ui/src/callWithMaskHandling.ts

```diff
@@ -17,9 +17,15 @@ export async function callWithMaskHandling<T>(callback: () => Promise<T>, valueT
             throw error;
         }
 
-        const maskedMessage: string = valueToMask ?
+        let maskedMessage: string = valueToMask ?
```

Comment:
You should be able to do this in a loop instead of copying code

Something like:
```
const valuesToMask: string[] = [originalValueToMask, encodeURIComponent(originalValueToMask)];
for (const valueToMask of valuesToMask) {
```